src/main/webapp/app/common/domain/JWTStoreService.ts

```diff
@@ -0,0 +1,31 @@
+import { defineStore } from 'pinia';
```

Comment:
we are in domain so we should not use pinia -> no framework

this file should not be here